robot/client/client.go

```diff
@@ -471,8 +471,10 @@ func (rc *RobotClient) checkConnection(ctx context.Context, checkEvery, reconnec
 	}
 }
 
-// Close cleanly closes the underlying connections and stops the refresh goroutine
+// Close the underlying connections and stop any periodic tasks across all constituent parts of the machine.
```

Comment:
```suggestion
// Close closes the underlying client connections to the machine and stops any periodic tasks running in the client.
```
I think here is a place where we should be specific about client/machine difference, reading the original change it could be interpreted as `any periodic tasks that are running in the actual machine` (e.g. if the machine gets a sensor reading every 10s) vs the correct `any periodic tasks that are running in the client to the machine`